README.md

```diff
@@ -1,143 +1,84 @@
-project:
-`bundle exec jekyll serve`
-This will build the site and start up a preview server that allows you to view the
-work that you have done.
-
-### Linux
-
-First, make sure you have Ruby v2.5.1 installed on your machine. There is a proven guide
-for getting Ruby installed on your machine that you can read
-[here](https://gorails.com/setup/ubuntu/16.04). This particular
-guide only needs to be followed up until the `Configuring Git` heading. Everything
-from `Configuring Git` onward is unnecessary to follow.
-Once you have Ruby v.2.5.1 installed on your machine, you are ready to build the project:
-`bundle exec jekyll serve`
-This will build the site and start up a preview server that allows you to view the
-work that you have done.
-
-## Contributing To The Repo
-
-As with most crdschurch repos, local feature development should be done against the `development` branch. Follow these simple steps to contribute:
-
-1. `git checkout development` and make sure that you are up to date (run `git pull`)
-2. Checkout a fresh branch based off of development: `git checkout -b feature/my-branch-name`
-3. Make local code changes and commit to your new branch
-4. When the time is right, create a pull request comparing your new branch to development
-5. After a review, your work will be merged into `development`
-   After these first 5 steps, there are 2 deployment possibilities: Standard or Expedited. See below and choose the right choice for your work.
-
-- Standard: Wait until the normal production deploy which takes place every 2 weeks on Wednesday afternoon. This is the best choice and should be used unless the work is extremely time sensative.
-- Expedited: Sometimes things just need to go to `prod` immediately. If this is the case follow the steps listed [in this doc](https://slack-files.com/T02C3F91X-FD2HHNMT7-c6400a04c1) or reachout to a Bearsharks team member.
-
-### Editing Meta Data (in the codebase)
-
-There will be times when you might want to provide custom meta data for page that lives in the repo (think /oakley). Simple add this block to your front matter above the last `---`:
-
-```yaml
-meta:
-  title: My Amazing Page
-  description: You won't believe how amazing this page is.
-  image:
-    url: "https://amazing-photos.net/cool"
-```
-
-**Note: indentation matters in `yaml`. Make sure you follow the pattern above.**
-
-## Requesting Content
-
-First, you need to export the following environment variables (you can get these
-values directly from Contentful)...
-
+## Table Of Contents
+1. [Quick Start](#quick-start)
+2. [Working With Content](#working-with-content)
+3. [Submodules](#submodules)
+4. [Advanced Setup](#advanced-setup)
+5. [Contributing To The Repo](#contributing-to-the-repo)
+6. [License](#license)
+
+## Quick Start
+1. Clone the repo: `git clone https://github.com/crdschurch/crds-net.git` then `cd ./crds-net`
+2. If you don't have Bundler/Jekyll, install it: `gem install bundler jekyll`
```

Comment:
Can you add in "make sure you have Ruby installed - check OS specific install instructions" between steps 1 and 2?